include/irr/asset/IMeshPacker.h

```diff
@@ -21,13 +18,96 @@ template <typename MeshBufferType>
 class IMeshPacker
```

Comment:
add a second template parameter to the list,  `MDIStructType` and use this instead of `DrawElementsIndirectCommand_t` (but you can static_assert that `is_base_of<DrawElementsIndirectCommand_t,MDIStructType>`)

this is to allow the user to add data onto the MDI parameter data stream, like this
```cpp
struct CustomDrawElementsIndirectCommand_t : DrawElementsIndirectCommand_t
{
   MyExtraData_t extraData;
};
```